pkgs/development/python-modules/zstd/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "zstd";
+  version = "1.3.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0d1ixayncfvikxckj78kr11l1h83zid04b7xfhf18ji8ykqz6fy0";
+  };
+
+  # No such file or directory: 'tests'
+  doCheck = false;
```

Comment:
This will need to be removed after the next release: https://github.com/sergey-dryabzhinsky/python-zstd/commit/3eb7b5d1299faaf8ca337607eac21c8dafcf92f2
Please do so.

Reply:
Tried to patch it from the fetchPypi src but didn't seem to work, test directory still not found. Building from github now.

Reply:
I think @dotlambda meant that if you are going to maintain the package and will do update to next stable version, you should not forget to remove `doCheck`. You can edit the comment above to remind yourself (or other contributor).

Reply:
That's exactly what I meant. You can stick to `fetchPypi` now (it's even preferrable because then we can auto-upgrade).

Reply:
@dotlambda using `fetchFromGitHub` allows us to run tests. I think, tests are very valuable in this case sine we would like to check that the package works against our zstd. So I would stick with `fetchFromGitHub` at least for this version.

Reply:
Alright. Then, @eadwu needs to remember to switch to `fetchPypi` with the next update.